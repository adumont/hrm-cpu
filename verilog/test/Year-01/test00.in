52 7d
15 30
46 92
40 a5
16 77
43 f7
33 01
