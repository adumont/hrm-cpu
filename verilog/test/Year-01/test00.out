7d
30
92
